contributors/devel/architectural-roadmap.md

```diff
@@ -935,7 +935,7 @@ features could be addressed by the following mechanisms:
   out into its own configuration
   object(s).](https://github.com/kubernetes/kubernetes/issues/19831)
   This should include the [feature-gate
-  mechanism](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/runtimeconfig.md).
+  mechanism](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/cluster-lifecycle/runtimeconfig.md).
```

Comment:
same repo. This can be:
`mechanism](/contributors/design-proposals/cluster-lifecycle/runtimeconfig.md)`